AzureMonitorAgent/manifest.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<ExtensionImage xmlns="http://schemas.microsoft.com/windowsazure">
+  <ProviderNameSpace>Microsoft.Azure.Monitor</ProviderNameSpace>
+  <Type>AzureMonitorLinuxAgent</Type>
+  <Version>1.4</Version>
+  <Label>Microsoft Azure Monitoring Agent for Linux</Label>
+  <HostingResources>VmRole</HostingResources>
+  <MediaLink></MediaLink>
+  <Description>Microsoft Azure Monitoring Agent for Linux</Description>
+  <IsInternalExtension>true</IsInternalExtension>
+  <Eula>https://msazure.visualstudio.com/DefaultCollection/One/_git/Compute-Runtime-Tux?path=%2FLICENSE</Eula>
```

Comment:
is this accessible outside of corp without MSFT creds?

Reply:
I don't assume so, but where else can we pull the License from? Should I add that licence into this repo?

Reply:
ok, can you point it to this doc for now ? https://docs.microsoft.com/en-us/azure/azure-monitor/learn/quick-collect-linux-computer . I am not sure if we should release the other one rt now

Reply:
Will do